storage/client/queue/README.md

```diff
@@ -184,22 +184,22 @@ The following sections provide several code snippets covering some of the most c
 - [Set/Update a Queue metadata](#Set-a-queue-metadata)
 
 ### Build a client
-We have two ways of building QueueService or Queue Client. Here will take queueServiceClient as an example. Same things apply to queueClient.
+We have two ways of building QueueStorage or Queue Client. Here will take queueStorageClient as an example. Same things apply to queueClient.
 
 First, build client from full URL/endpoint (e.g. with queueName, with SASToken and etc.)
 
 ```Java
-String queueServiceURL = String.format("https://%s.queue.core.windows.net/%s", accountName, sasToken);
-QueueServiceClient queueServiceClient = new QueueServiceClientBuilder().endpoint(queueServiceURL).buildClient();
+String queueStorageURL = String.format("https://%s.queue.core.windows.net/%s", accountName, sasToken);
+QueueStorageClient queueStorageClient = new QueueStorageClientBuilder().endpoint(queueStorageURL).buildClient();
 ```
 
 Or
 
-We can build the queueServiceClient from the builder using `${SASToken}` as credential.
+We can build the queueStorageClient from the builder using `${SASToken}` as credential.
 
 ```Java
-String queueServiceURL = String.format("https://%s.queue.core.windows.net", accountName);
-QueueServiceClient queueServiceClient = new QueueServiceClientBuilder().endpoint(queueServiceURL).credential(SASToken).buildClient();
+String queueStorageURL = String.format("https://%s.queue.core.windows.net", accountName);
```

Comment:
I'm not really understanding the explicit differences in these examples, instead change this to show that the endpoint method is able to parse append SAS token and use them as authentication.

Reply:
Added comments to achieve the purpose.

The example is to show how to build the client. The example of showing how endpoint pass in the authentication is a little underneath, I don't think it is needed for the readme.